pkg/specgen/generate/namespaces.go

```diff
@@ -454,7 +454,7 @@ func GetNamespaceOptions(ns []string, netnsIsHost bool) ([]libpod.PodCreateOptio
 	for _, toShare := range ns {
 		switch toShare {
 		case "cgroup":
-			options = append(options, libpod.WithPodCgroups())
+			options = append(options, libpod.WithPodCgroup())
```

Comment:
`WithPodCgroup` or `Cgroups`?

Reply:
I think this should be `WithPodCgroup`, which sets `pod.config.UsePodCgroupNS` to `True`. It might be better to rename these functions to make their roles clear.